tests/test_modifiers.py

```diff
@@ -1,15 +1,15 @@
 import pyhf
 import numpy
 
-modifiers_to_test = [
-    "histosys",
-    "normfactor",
-    "normsys",
-    "shapefactor",
-    "shapesys",
-    "staterror",
-]
-modifier_pdf_types = ["normal", None, "normal", None, "poisson", "normal"]
+# modifiers_to_test = [
+#     "histosys",
+#     "normfactor",
+#     "normsys",
+#     "shapefactor",
+#     "shapesys",
+#     "staterror",
+# ]
+# modifier_pdf_types = ["normal", None, "normal", None, "poisson", "normal"]
```

Comment:
These don't seem to be used anywhere, but maybe should get removed in a separate cleaning PR.

Reply:
Oh they should have gotten removed in PR #1625 but got missed by all of us.